res/css/views/rooms/_EditMessageComposer.scss

```diff
@@ -16,13 +16,13 @@ limitations under the License.
 */
```

Comment:
The description only includes an after screenshot which makes it hard to tell what changed. Can we add a before and after screenshot for easy comparison?

A bulleted text summary would also be good to check off in your mind as you look at the screenshots.

Reply:
I've updated the description

Reply:
@SimonBrandner It looks like it's two different sections of a room which threw me off for a bit. The viewport could also be a lot narrower in these cases since we're only trying to show off the message editing form which has a bunch of dead space which is cumbersome to compare big images. I realize aspect ratio of components is sometimes important for full app context and feel but could definitely be a lot narrower here.

Thanks for the text summary 👍